app/models/cdr_export.rb

```diff
@@ -19,10 +19,20 @@ class CdrExport < Yeti::ActiveRecord
   STATUS_PENDING = 'Pending'.freeze
   STATUS_COMPLETED = 'Completed'.freeze
   STATUS_FAILED = 'Failed'.freeze
+  STATUSES = [
+    STATUS_PENDING,
+    STATUS_COMPLETED,
+    STATUS_FAILED
+  ].freeze
+
+  #need for activeadmin form
+  attr_accessor :customer_acc_id_eq,
+    :is_last_cdr_eq, :time_start_gteq, :time_start_lteq
 
   validates_presence_of :status, :fields, :filters
   validate do
-    if filters.keys.exclude?('time_start_lteq') || filters.keys.exclude?('time_start_gteq')
+    if filters.keys.exclude?('time_start_lteq') || filters.keys.exclude?('time_start_gteq') ||
+        filters['time_start_gteq'].empty? || filters['time_start_lteq'].empty?
```

Comment:
@gigorok  I think it can be simplier just
 ```filters['time_start_gteq'].empty? || filters['time_start_lteq'].empty?```

Reply:
agree